playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -8,3 +8,5 @@
       role: common
       local_facts:
         hostname: "{{ openshift_hostname | default(None) }}"
+  - set_fact:
+      openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
```

Comment:
Oh right, except for the complication of the openshift_docker_hosted_registry_network which has to be based on the first master. I will leave that item here and we can address with delegate_to perhaps in the future.